Windows CI build always fails

Windows CI build always fails with the following error:

~~~
Run deploy_tool="windeployqt-qt6.exe"
D:\a\_temp\3e9a044a-7d69-481b-a4d1-168fc8f1aeab: line 12: windeployqt-qt6.exe: command not found
Error: Process completed with exit code 127.
~~~

It seems that something has changed and windeploy-qt6 tool is missing in Qt6 installation.  

@dsm , could you look at this?

--- a/.github/workflows/deploy.yml
+++ b/.github/workflows/deploy.yml
@@ -431,7 +431,7 @@ jobs:
 
     - name: Deploy Qt6 dependencies
       run: |
-        deploy_tool="windeployqt-qt6.exe"
+        deploy_tool="windeployqt.exe"
         bin_dir="build/qucs-suite/bin"
         options="--svg --no-translations --no-system-d3d-compiler --no-network"
 

